src/components/Team/CardContainer.jsx

```diff
@@ -68,306 +43,166 @@ const Legend = styled.p`
-                            gradYear="2020"
-                            interests="Rap music, being stoked, tennis"
-                            funFact="Last name means “cricket” in Slovakian"
-                            githubUrl="https://github.com/csvrcek"
-                            instagramUrl="https://www.instagram.com/cdids/"
-                            linkedinUrl="https://www.linkedin.com/in/connor-svrcek-aab07612b/"
-                            facebookUrl="https://www.facebook.com/connor.svrcek"
-                            personalUrl="https://connorsvrcek.com/"
-                            imagePath={Connor} />
-                    </div>
-                    <div>
-                        <MemberCard name="Trevor Sanderson" title="Senior Advisor"
-                            gradYear="2019"
-                            interests="Video Games, Fantasy Football, Rap music"
-                            funFact=""
-                            githubUrl="https://github.com/tjsande"
-                            linkedinUrl="https://www.linkedin.com/in/trevor-sanderson-459028171/"
-                            imagePath={Trevor} />
+const LEADERSHIP_FIELD_TO_COL = {
+    name: 0,
+    uniqname: 1,
+    title: 2,
+    gradYear: 3,
+    interests: 4,
+    funFact: 5,
+    linkedinUrl: 6,
+    githubUrl: 7,
+    personalUrl: 8,
+    facebookUrl: 9,
+    instagramUrl: 10,
+    imageUrl: 11,
+    display: 12
+};
+
+class CardContainer extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            leadership: [],
+            categories: []
+        };
+        this.getLeadership = this.getLeadership.bind(this);
+    }
+
+    getLeadership() {
+        const params = {
+            spreadsheetId: process.env.REACT_APP_LEADERSHIP_ID,
+            range: "Sheet1!A1:AA998",
+        };
+         
+        gapi.client.sheets.spreadsheets.values.get(params)
+            .then(response => {
+                const categories = [];
+                // Do a pass to find the categories. Categories must be nonempty.
+                response.result.values.forEach((row, index) => {
+                    if (row[0] === "CATEGORY") {
+                        categories.push({
+                            name: row[1],
+                            startIndex: index
+                        });
+                    }
+                });
+                this.setState({
+                    categories,
+                    leadership: response.result.values
+                });
+            }, error => {
+                console.log("error: " + error.result.error.message);
+            });
+    }
+
+    componentDidMount() {
+        const gapiScript = document.createElement('script');
+        gapiScript.src = 'https://apis.google.com/js/api.js?onload=onGapiLoad';
+        document.body.appendChild(gapiScript);
+        window.onGapiLoad = () => {
```

Comment:
Yes, this is scuffed. But it's the only way to load this JS, which is the only API I could find with decent documentation. Trust me, I tried other things

Reply:
yeah, i don't doubt you'd do this only as a last resort